R/widerRepeated.R

```diff
@@ -0,0 +1,63 @@
+#' @name widerRepeated
+#' @title Transform Data Into Wide Format
+#' 
+#' @description Converts a dataframe into wide format given a single REDCap form.
+#' This function assumes that the Records argument is the result of exportRecordsTyped,
+#' and that all empty values have been previously dropped. This will only widen dataframes 
+#' that have a unique identification variable (e.g. 'record_id'), "redcap_event_name" and 
+#' "redcap_repeat_instrument" in the fields. Otherwise, the data passed in will be returned 
+#' unchanged.
+#'   
+#' @param Records A \code{data.frame} containing the records from
+#'        \code{\link{exportRecordsTyped}}
+#' @param rcon A REDCap connection object as created by \code{unlockREDCap}.
+#' @export
+widerRepeated <- function(Records, rcon)
+{
+  idvar <- getProjectIdFields(rcon)
+  ###########################################################################
+  # Check parameters passed to function
+  coll <- checkmate::makeAssertCollection()
+  
+  checkmate::assert_data_frame(x = Records, 
+                               add = coll)
+  checkmate::assert_class(x = rcon,
+                          classes = "redcapApiConnection",
```

Comment:
`classes = "redcapConnection"`

This will allow the function to work with offline connections as well.